src/lib/auth.ts

```diff
@@ -56,6 +56,15 @@ export const ROLES: {
 		'resin.device_environment_variable.all',
 		'resin.device_tag.all',
 		'resin.device_service_environment_variable.all',
+
+		// These are for the supervisor proxy
+		'resin.device.purge',
+		'resin.device.shutdown',
+
+		// These are for the actions proxy
```

Comment:
These may not be necessary to add here as I believe open-balena users do not have the actions proxy anyway

Reply:
Indeed I think you can drop these